crates/djangofmt_lint/tests/check/form_method/form_method.valid.html

```
@@ -0,0 +1,19 @@
+<!-- Valid methods -->
+<form method="get">Submit</form>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)